docs/astro/src/content/docs/reference/primitive-types.mdx

```diff
@@ -146,8 +146,10 @@ Slint looks for images in the following places:
 
 Loading image from `http` is only supported in [SlintPad](https://slintpad.com).
 
-Supported format are SVG, as well as formats supported by the [`image` crate](https://crates.io/crates/image):
-AVIF, BMP, DDS, Farbfeld, GIF, HDR, ICO, JPEG, EXR, PNG, PNM, QOI, TGA, TIFF, WebP.
+Supported format are SVG, PNG, and JPEG.
+Other formats supported by the [`image` crate](https://crates.io/crates/image)
+(AVIF, BMP, DDS, Farbfeld, GIF, HDR, ICO, JPEG, EXR, PNG, PNM, QOI, TGA, TIFF, WebP)
+can be enabled using the `image-default-formats` compilation feature.
```

Comment:
This is Rust specific advice in what's a generic section. I think this could either be solved using tabs, two paragraphs:

One that explains that for Rust, not all formats are enabled by default and how to enable them.

A second one that for all other programming languages, all image formats are enabled by default.

Reply:
```suggestion
Supported format are SVG, and formats supported by the [`image` crate](https://crates.io/crates/image): 
AVIF, BMP, DDS, Farbfeld, GIF, HDR, ICO, JPEG, EXR, PNG, PNM, QOI, TGA, TIFF, WebP.

For Rust applications, not all formats are enabled by default and can be enabled with the `image-default-formats` compilation features
```